tests/manage/ocs_288.py

```diff
@@ -0,0 +1,292 @@
+"""
+A test for creating pvc with random sc
+"""
+import logging
+import os
+import random
+
+import pytest
+
+from ocs import defaults, constants
+from ocs import ocp
+from ocsci.config import ENV_DATA
+from ocsci.testlib import tier1, ManageTest
+from resources import pod
+from resources.ocs import OCS
+from tests import helpers
+
+log = logging.getLogger(__name__)
+TEMPLATE_DIR = "templates"
+
+RBD_POOL_YAML = os.path.join(
+    "templates/ocs-deployment", "cephblockpool.yaml"
+)
+
+TEMP_YAML_FILE = 'temp.yaml'
+
+SC = ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+POOL = ocp.OCP(
+    kind='CephBlockPool', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+POD = ocp.OCP(
+    kind='Pod', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+CFS = ocp.OCP(
+    kind='CephFilesystem', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+
+@pytest.fixture(scope='class')
+def ocs288_fixture(request):
+    def finalizer():
+        teardown()
+
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def setup():
+    """
+    Setting up the environment for the test
+    """
+    log.info("Creating RBD Secret")
+    global RBD_SECRET_OBJ
+    RBD_SECRET_OBJ = helpers.create_secret(constants.CEPHBLOCKPOOL)
+
+    log.info("Creating CEPHFS Secret")
+    global CEPHFS_SECRET_OBJ
+    CEPHFS_SECRET_OBJ = helpers.create_secret(constants.CEPHFILESYSTEM)
+
+    log.info("Creating CEPH FileSystem")
+    assert create_cephfilesystem()
+
+    log.info("Creating RBD Storageclass")
+    assert create_multiple_rbd_storageclasses(count=5)
+
+    log.info("Creating CEPHFS Storageclass")
+    assert create_storageclass_cephfs()
+
+
+def teardown():
+    """
+    Tearing down the environment
+    """
+    global RBD_SECRET_OBJ, CEPHFS_SECRET_OBJ, CEPHFS_OBJ
+    log.info("Deleting PVC")
+    assert delete_pvc()
+    log.info("Deleting CEPH BLOCK POOL")
+    assert delete_cephblockpool()
+    log.info("Deleting RBD Secret")
+    RBD_SECRET_OBJ.delete()
+    log.info("Deleting CEPHFS Secret")
+    CEPHFS_SECRET_OBJ.delete()
+    log.info("Deleting CEPH FILESYSTEM")
+    CEPHFS_OBJ.delete()
+    log.info("Deleting Storageclass")
+    assert delete_storageclass()
+
+
+def delete_pvc():
+    """
+    Function to delete pvc
```

Comment:
fix docstring.

No need for 
:param pvc_list:
￼     :return:

Reply:
Done